src/app/ui/ui-select/ui-select.component.html

```diff
@@ -17,5 +17,6 @@
         {{getLabel(value)}}
       </a>
     </li>
+    <li *ngIf="bottomLabel" class="bottom-label"><a class="dropdown-item">Zoom for more</a></li>
```

Comment:
Should `{{bottomLabel}}` be in the dropdown-item, instead of `Zoom for more`?  Just in case we want different messages in other dropdowns.

Reply:
Ah whoops, that's a typo, I'll fix that